new-frontend/src/app/components/loginForm.tsx

```diff
@@ -0,0 +1,21 @@
+import {
+  FormControl,
+  FormLabel,
+  Input,
+  Heading,
+  Button,
+  Box
+} from '@chakra-ui/react'
+
+export const LoginForm = () => {
+  return (
+    <FormControl isRequired>
+      <Box gap='20px' maxW='md' height='60vh' borderWidth='1px' p='50px' borderRadius='md'>
+        <Heading>User Login</Heading>
+        <FormLabel>username</FormLabel>
+        <Input placeholder='enter your username' />
```

Comment:
```suggestion
<FormControl>
        <FormLabel>username</FormLabel>
        <Input placeholder='enter your username' />
</FormControl>
```

It would be nice if we add the `FormControl` [component](https://v2.chakra-ui.com/docs/components/form-control) too. It provides the context for both label and input and connect's them.